pkg/kubelet/dockershim/docker_container_windows.go

```diff
@@ -54,7 +53,7 @@ func (ds *dockerService) applyPlatformSpecificDockerConfig(request *runtimeapi.C
 	return cleanupInfo, nil
 }
 
-// applyGMSAConfig looks at the kuberuntime.GMSASpecContainerAnnotationKey container annotation; if present,
+// applyGMSAConfig looks at the container's `.Windows.SecurityContext.GmsaCredentialSpec` field; if present,
```

Comment:
nit: I think the single quotes around `.Windows.SecurityContext.GmsaCredentialSpec` can be removed from the comments.